### ISSUE TITLE
PSR12 not handling multiple namespace blocks correctly

### ISSUE THREAD

**Issue by kwhat** [squizlabs/PHP_CodeSniffer#2854]
Hi,

It seems that multi-line namespace blocks are complaining about a non-existent newline issue.

```
FILE: ...runner/work/requestful/requestful/tests/Unit/Http/ClientTest.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 161 | ERROR | [x] Header blocks must not contain blank lines
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```
PHPCBF cannot fix the issue.  The file in question is here: https://github.com/kwhat/requestful/blob/master/tests/Unit/Http/ClientTest.php

Thanks

**Comment by gsherwood** [comment]
Sorry, missed this. It was later reported with a simple code snippet to replicate, so closing this one in favour of the other (issue #2917). Will use your sample code when testing a fix though. Thanks for the report.
